engines/e_afalg.c

```diff
@@ -355,13 +355,34 @@ static int afalg_fin_cipher_aio(afalg_aio *aio, int sfd, unsigned char *buf,
                         }
                         continue;
                     } else {
+                        char strbuf[32];
+                        /* 
+                         * sometimes __s64 is defined as long long int
+                         * but on some archs ( like mips64 or powerpc64 ) it's just long int
+                         * to be able to use BIO_snprintf() with PRId64 without warnings
+                         * copy events[0].res to an int64_t variable
+                         * because __s64 and int64_t should always be a 64 bit type this should work
+                         */
+                        int64_t op_ret = events[0].res;
+
                         /*
                          * Retries exceed for -EBUSY or unrecoverable error
                          * condition for this instance of operation.
                          */
                         ALG_WARN
                             ("%s(%d): Crypto Operation failed with code %lld\n",
                              __FILE__, __LINE__, events[0].res);
+                        BIO_snprintf(strbuf, sizeof(strbuf), "%"PRId64, op_ret);
```

Comment:
that does not work because on %lld is for long long int

on some systems ( mips64, powerpc64 ) a 64 bit integer is defined as long int and it must be %ld

the cross compiler checks fail with %lld

Reply:
> you can allow me to run the github actions ?

Unfortunately I cannot.

> on some systems ( mips64, powerpc64 ) a 64 bit integer is defined as long int and it must be %ld

Weird that we have other BIO_snprintf calls with lld used for int64_t. Anyway just cast the op_ret to long long int.

Reply:
ok changed it to %lld

lets see if the cross-compiler check work now

Reply:
ok next try

this time i use a long long int variable and %lld

mybe this works